### PR TITLE
docs(release-notes): release-notes changes for 1.1 release

### PR DESCRIPTION
**Pull request by matt-primrose**
## PR Checklist

- [x] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [x] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?
Updates to release-notes to describe feature and issue changes for 1.1 release

## Anything the reviewer should know when reviewing this PR?
Trying to keep this higher level than the changelog for each repository

### CODE REVIEW

**Comment by bwendlandt-intel** [comment]
Just a general thought: Do we want this to be a running doc rather than just latest? Keep all release notes listed in sections by version from latest descending?

File: docs/release-notes.md [hunk 1]
```diff
@@ -1,53 +1,39 @@
...
+- **CORS support:** Added support for Cross-Origin Resource Sharing (CORS).
 
 #### [RPC](Glossary.md#r)
-- **microLMS in the RPC executable:** Eliminates the need to run a separate LMS when trying to configure Intel® AMT.
-- **CentOS*:** Enables CentOS on RPC. Linux build environment is still Ubuntu* 18.04.
-- **Intel® AMT information commands:**  Provide feedback to the local command prompt showing status information of the Intel® AMT device.
+- **Improved Link Status when using --amtinfo:** Fixed link status not being set.
+- **Add/update DNS Suffix (OS), Hostname (OS), FQDN (AMT), and DNS Suffix (AMT):** Improved pulling metadata from Intel&reg; AMT and Host OS.
+- **Improved build scripts for Windows and Linux:**  Build scripts have been updated so that the build time and executable size is more consistent between Windows and Linux.
+- **CentOS 7 Support:** Build steps added to support CentOS 7.
```
Comment (trleasher-intel) [inline docs/release-notes.md:15]: Should add a bullet for CentOS 8 support as well, since the documentation calls it out specifically.

**Review by trleasher-intel — APPROVED** [review]

File: docs/release-notes.md [hunk 2]
```diff
@@ -1,53 +1,45 @@
...
+- **CORS support:** Added support for Cross-Origin Resource Sharing (CORS).
 
 #### [RPC](Glossary.md#r)
-- **microLMS in the RPC executable:** Eliminates the need to run a separate LMS when trying to configure Intel® AMT.
-- **CentOS*:** Enables CentOS on RPC. Linux build environment is still Ubuntu* 18.04.
-- **Intel® AMT information commands:**  Provide feedback to the local command prompt showing status information of the Intel® AMT device.
-  
+- **Added/updated getting metadata from Intel&reg; AMT device:** Improved getting DNS Suffix (OS), Hostname (OS), FQDN (AMT), and DNS Suffix (AMT) from Host OS and Intel&reg; AMT.
+- **CentOS 7 Support:** Build steps added to support CentOS 7.
+- **--nocertcheck option:** Option added to skip the TLS cert check for Websocket connections. This is to aid testing in development environments where self-signed TLS certificates are used.
```
Comment (rsdmike) [inline docs/release-notes.md:14]: I would state how this previously worked. like: "Previously, this required a debug version of RPC" or something to that effect.

**Review by rsdmike — APPROVED** [review]

File: docs/release-notes.md (hunk 1 above)
Comment (matt-primrose) [reply docs/release-notes.md:13]: We had CentOS support in 1.0 release notes.  No version called out specifically, but yes, it was CentOS 8.  1.1 adds support for CentOS 7.

File: docs/release-notes.md (hunk 2 above)
Comment (matt-primrose) [reply docs/release-notes.md:14]: That was an undocumented feature.  Now it's documented. 🥇

**Pull request merged**